spec/integration/util/windows/user_spec.rb

```diff
@@ -117,6 +117,28 @@ def expect_logon_failure_error(&block)
       it "should return false given a nil username and an incorrect password" do
         expect(Puppet::Util::Windows::User.password_is?(nil, bad_password)).to be_falsey
       end
+
+      context "with a correct password" do
+        it "should return true even if account restrictions are in place " do
+          Puppet::Util::Windows::User.stubs(:logon_user).raises(Puppet::Util::Windows::Error.new('', 1327))
```

Comment:
Hmm, I'm not completely sure about 1907 or 1330, because both may be valid, but the user/password combination should sync instead of returning true in those cases. So letting them be false to me seems like the right path forward. Or what we are saying for negative tests, that's cool as well. 👍